src/Util/FundingOrgFilter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Util;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+use App\Entity\FundingOrganization;
+
+/**
+ * A utility to determine if and which Funding Organizations need to be filtered by.
+ */
+class FundingOrgFilter
+{
+    /**
+     * The entity manager to use.
+     *
+     * @var EntityManagerInterface
+     */
+    protected $entityManager;
+
+    /**
+     * An array of the funding organization to filter by.
+     *
+     * @var array
+     */
+    protected $fundingOrganizations;
+
+    /**
+     * Constructor.
+     *
+     * @param EntityManagerInterface $entityManager The entity manager to use.
+     * @param array                  $fundingOrgs   Funding Organizations to filter by.
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        array $fundingOrgs
+    ) {
+        $this->entityManager = $entityManager;
+
+        $this->shortNamesToIdArray($fundingOrgs);
+    }
+
+     /**
+     * Set array of FOs to filter by.
+     *
+     * @param array $fundingOrgs List of Funding Organizations Short Names.
+     *
+     * @return void
+     */
+    private function shortNamesToIdArray(array $fundingOrgs)
```

Comment:
This method is being called only once (from the constructor) and its a one liner, it could stay in the constructor itself